database/frontend/images.json

```diff
@@ -25,5 +25,20 @@
         "url": "https://kaboompics.com/",
         "category": "frontend",
         "subcategory": "images"
```

Comment:
This could use a description (e.g., the benefits of using this tool). What do you think @rupali-codes?

Reply:
yeah, but since it's for images, what could be the benefits? 😅